paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass.cc

```diff
@@ -177,7 +177,12 @@ void CPUQuantizeSquashPass::ConvDequantSquash(Graph* graph) const {
     GET_IR_NODE_FROM_SUBGRAPH(dequant_out, dequant_out, conv_dequant_pattern);
 
     // if conv2d has one output
-    if (conv_out->outputs.size() == 1) {
+    // and there is no fuse residual connection
+    // because residual fusion does not support force output with fp32
+    if (conv_out->outputs.size() == 1 &&
+        !(conv_op->Op()->HasAttr("fuse_residual_connection") &&
```

Comment:
Is it necessary to perform conv out number check?

Also, you can replace the second condition with `!(conv_op->GetAttrIfExists("fuse_residual_connection"))`

Reply:
Yes, we should check conv out number if its one or check if every op after conv is dequantize. 
Right now there is no model that have that, but its good to avoid such situation. 
![image](https://user-images.githubusercontent.com/33838455/65131814-2eb54e80-da00-11e9-938c-c0dae7b7eb96.png)